### DISCUSSION TITLE
Support for Bengali

### DISCUSSION THREAD

**Discussion by nafeesmahbub** [coqui-ai/TTS#1508] [General Q&A]
Need some guidance to generate model for Bengali language.
I have trained a model with around 5 Hrs of data. The inference is pretty good.
But some words sound like robotic. 
For example, the numbers in Bengali sound robotic to be specific.
How to make certain word that sound robotic make good inference or should I have to train with more data.
Also is use_phoneme==true will increase the accuracy.

**Comment by p0p4k** [comment]
A 'phonemizer' converts written text into text that is read (vocal sounds). Check [this](TTS/tts/utils/text/japanese/phonemizer.py) file, a similar function for your language can be used. For example, a quick google search produced [this](https://github.com/rahmanabir/BanglaNLP/blob/master/data_dekhabet.py).
